qt/widgets/common/inc/MantidQtWidgets/Common/FitScriptGeneratorPresenter.h

```diff
@@ -66,7 +66,7 @@ class EXPORT_OPT_MANTIDQT_COMMON FitScriptGeneratorPresenter : public IFitScript
 
   void setWorkspaces(QStringList const &workspaceNames, double startX, double endX);
   void addWorkspaces(std::vector<Mantid::API::MatrixWorkspace_const_sptr> const &workspaces,
-                     std::vector<WorkspaceIndex> const &workspaceIndices);
+                     std::vector<WorkspaceIndex> const &spectraIndices);
```

Comment:
Can we leave this as `workspaceIndices` too as this will also reduce variable naming consistency in this file